.github/workflows/tests.yml

```diff
@@ -5,11 +5,26 @@ on: [push, pull_request]
 jobs:
   test:
     runs-on: ubuntu-latest
+    services:
```

Comment:
This creates a Postgres container to spin up a database to test the database integration.